Dockerfile-pdf-testing

```diff
@@ -1,15 +1,23 @@
-FROM cypress/base:14.19.0
+FROM cypress/base:16.17.1
```

Comment:
Are we upgrading this everywhere? The base image version is tied to the version of Node that we are on, which is 14.19.0.

Reply:
Yes, we are also in the process of upgrading to Node 16 everywhere.  We are taking on the Devex task next.